docs/tutorial/flask.rst

```diff
@@ -329,6 +326,12 @@ look like the following:
 Since you are creating a new version of your application, **open the**
 ``rockcraft.yaml`` file and change the ``version`` (e.g. to ``0.2``).
 
+.. note::
+
+    ``rockcraft pack`` will create a new image with the updated code even if you
+    don't change the version. It is recommended to change the version whenever
+    you make changes to the application in the image.
```

Comment:
Bumping the version number in the tutorial is good because it demos how software releases integrate with rockcraft, and demos the chisel feature, but I think some may interpret this as "whenever changes are made during development" and not just when software releases are made.

Reply:
Perhaps something like this

``` markdown
    ``rockcraft pack`` will overwrite any existing image with the same filename. Updating the version field in the `rockcraft.yaml` will automatically update the filename upon packing, preventing conflicts with prior image versions.
```

Reply:
Hmm, that makes it sound like the motivation is around keeping files around, do you think it is more related to releasing?

Reply:
I would argue in the tutorial its more about keeping the images so we can compare the chiseled and unchiseled images, but in the context of software development I would only expect the software version to be bumped occasionally.